storybook/cypress/e2e/button-group.cy.ts

```diff
@@ -0,0 +1,14 @@
+describe("Button Group", () => {
```

Comment:
Voor nu hele basic tests, dat er iets is. Mocht dit component over 'n tijdje wat prominenter worden gebruikt breiden we het wel uit.